client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -83,17 +84,22 @@ class RegistrantExtraInfoFrForm extends React.PureComponent {
 		//    fields so we can keep them together in one place
 		defaultRegistrantType = this.props.contactDetails.organization ? 'organization' : 'individual';
 
-		this.props.updateContactDetailsCache( {
+		const payload = {
 			extra: {
 				fr: { registrantType: defaultRegistrantType },
 			},
-		} );
+		};
+
+		this.props.updateContactDetailsCache( payload );
+		this.props?.onContactDetailsChange( payload );
```

Comment:
You probably meant `this.props.onContactDetailsChange?.(payload)` since `this.props` will always exist.